src/server/license/cmds/cmds.go

```diff
@@ -77,20 +64,14 @@ func ActivateCmd() *cobra.Command {
 				return errors.Wrapf(inspectErr, "could not inspect cluster")
 			}
 
-			// inspect the active context to determine whether its pointing at an enterprise server
-			enterpriseServer, err := getIsActiveContextEnterpriseServer()
-			if err != nil {
-				return err
-			}
-
 			// Register the localhost as a cluster
 			resp, err := c.License.AddCluster(c.Ctx(),
 				&license.AddClusterRequest{
 					Id:                  "localhost",
 					Address:             "grpc://localhost:653",
 					UserAddress:         "grpc://localhost:653",
 					ClusterDeploymentId: clusterInfo.DeploymentID,
-					EnterpriseServer:    enterpriseServer,
+					EnterpriseServer:    true,
```

Comment:
note: whatever cluster the license is activated on should note itself as an enterprise server